README.rst

```diff
@@ -0,0 +1,16 @@
+##########
+pipe_tasks
+##########
+``pipe_tasks`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+
+``pipe_tasks`` provides many of the `Task <https://pipelines.lsst.io/py-api/lsst.pipe.base.Task.html#lsst.pipe.base.Task>`_ classes that drive the LSST Science Pipelines. The `command-line tasks listed here <https://pipelines.lsst.io/modules/lsst.pipe.tasks/index.html#lsst-pipe-tasks-command-line-tasks>`_ are useful data processing entry points for most users. These scripts can be found in the ``bin.src/`` folder.
```

Comment:
I think we should mention why they are in `bin.src/` instead of `bin/` (i.e., when built and installed they are in `bin/`).